sources/packages/forms/src/form-definitions/studentaccountactivity.json

```diff
@@ -530,6 +530,28 @@
             }
           ]
         },
+        {
+          "title": "Code PTD - Account Closed by Province",
```

Comment:
It is hard to define the pattern for the title of these contents.
There is an AC that states the title as `PTD - Account closed by province` but I am not sure if the intention was also to be the content title.
I would recommend either following the AC or asking biz.

Reply:
Updated AC to be explicit about title in form definition.